Node/vue-browser/tests/unit/example.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { mount } from '@vue/test-utils'
+import HelloWorld from '@/components/HelloWorld.vue'
+import axeDevtools from '@axe-devtools/browser';
+import Reporter from '@axe-devtools/reporter'
+
+
+const fixture = document.createElement('div');
+document.body.appendChild(fixture);
+// let reporter;
+describe('HelloWorld.vue', () => {
+  let reporter;
+  
+  beforeEach(() => {
+    reporter = new Reporter('A11yResults', './a11y-results')
+    })
+
+  afterEach(() => {
+    reporter.buildHTML('./a11y-results')
+    reporter.buildCSV('./a11y-results')
+    reporter.buildJUnitXML('./a11y-results')
+    })
```

Comment:
nit - before / after each remove the appended `fixture`, if we ever decide to add more tests, the fixture will stay mounted to the DOM

Reply:
Done